docs/setup_qa_pipeline.md

```diff
@@ -0,0 +1,57 @@
+# Steps to setup an end2end QA pipeline 
+Clone the Anserini, models, and data repo:
+```
+git clone https://github.com/rosequ/Anserini.git
+git clone https://github.com/castorini/data.git
+git clone https://github.com/castorini/models.git
```

Comment:
@nims11 only Anserini has python code and needs to be pip installed. @tuzhucheng clone the original Anserini repo and check if the issue repeats. I'll have the docs updated.